src/libbson/libbson.rc.in

```diff
@@ -29,10 +31,10 @@ BEGIN
             VALUE "FileDescription", "A BSON Library for C"
             VALUE "FileVersion", BSON_VERSION_S
             VALUE "InternalName", "bson-1.0"
-            VALUE "OriginalFilename", "bson-1.0.dll"
+            VALUE "OriginalFilename", BSON_OUTPUT_BASENAME "-1.0.dll"
```

Comment:
I think the instances of `1.0` should be replaced with `BSON_API_VERSION` for consistency with [OUTPUT_NAME](https://github.com/mongodb/mongo-c-driver/blob/1f8bf52e3b85324aab20503e60b0b129fd66ffd5/src/libbson/CMakeLists.txt#L250). Similarly for libmongoc.